doc/build-helpers/fixed-point-arguments.chapter.md

```diff
@@ -0,0 +1,89 @@
+# Fixed-point arguments of build helpers {#chap-build-helpers-finalAttrs}
+
+As mentioned in the beginning of this part, `stdenv.mkDerivation` could alternatively accept a fixed-point function. The input of such function, typically named `finalAttrs`, is expected to be the final state of the attribute set.
+A build helper like this is said to accept **fixed-point arguments**.
```

Comment:
> FWIW, the documentation currently calls those functions “explicitly recursive attribute sets” rather than “fixed-point functions/arguments.” I believe it would be better to be consistent, whichever term is prefered.

IIRC, we intentionally avoid the wording "recursive attribute set", as it refers to the anti-pattern we try to replace (`rec { ... }`).

The new name "fixed-point arguments" is decided here: https://github.com/NixOS/nixpkgs/pull/262648#discussion_r1368098933

> Of course, the prefered wording should be uniformized across the documentation (or at least across the new text in this PR)

I thought I had replaced them all in #262648. Is there something I forgot to change?